qiskit/schemas/backend_configuration_schema.json

```diff
@@ -35,7 +58,6 @@
                 "basis_gates": {
                     "type": "array",
                     "description": "List of basis gates names on the backend",
-                    "minItems": 1,
```

Comment:
I'm not sure I agree with this getting removed. Even simulators need to provide a setup of gates

Reply:
Consider the case with a raw device. It will not have any built-in gates. Pulse design techniques would then be used on top of OpenPulse to design the gate at which point the configuration would be updated with the gate.

Reply:
I think that's an edge case and I'm hesitant to reduce the ability to validate based on that. These get ignored when running in openpulse and so in the case of a raw device they would just need to provide some dummy gates. Although I don't there would ever be a true raw device offered (ie. that hasn't had at least a zero order cal)

Reply:
Ok! I will update the schemas here. I will also add dummy gates in the spec so that the spec tests continue to pass.